crates/wysiwyg/src/composer_model.rs

```diff
@@ -238,6 +242,47 @@ where
         )
     }
 
+    fn create_list(&mut self, ordered: bool) -> ComposerUpdate<C> {
+        // Store current Dom
+        self.push_state_to_history();
+        let list_tag = if ordered { "ol" } else { "ul" };
+        let (s, e) = self.safe_selection();
+        let range = self.state.dom.find_range(s, e);
+        match range {
+            Range::SameNode(range) => {
+                let node =
+                    self.state.dom.lookup_node(range.node_handle.clone());
+                if let DomNode::Text(t) = node {
+                    let text = t.data();
+                    let list_node = DomNode::new_list(
+                        list_tag.to_html(),
+                        vec![DomNode::Container(ContainerNode::new_list_item(
+                            "li".to_html(),
+                            vec![DomNode::Text(TextNode::from(text.to_vec()))],
+                        ))],
+                    );
+                    self.state.dom.replace(range.node_handle, vec![list_node]);
+                }
```

Comment:
We should panic in an else clause here - if it's not a text node, something is wrong.